### ISSUE TITLE
Is the project DEAD???

### ISSUE THREAD

**Issue by cstsiushkevich** [supasate/connected-react-router#613]
No any developer activity more than a year.

**Comment by Mehdi-HAFID** [comment]
yep, you can switch to [https://github.com/salvoravida/redux-first-history](https://github.com/salvoravida/redux-first-history)

**Comment by cstsiushkevich** [comment]
> yep, you can switch to https://github.com/salvoravida/redux-first-history

Thanks!
